x-pack/plugins/observability_solution/logs_explorer/public/controller/create_controller.ts

```diff
@@ -93,7 +96,7 @@ export const createLogsExplorerControllerFactory =
       customizations,
       datasetsClient,
       discoverServices,
-      event$: EMPTY,
+      event$: publicEvents$,
```

Comment:
I've decided to re-use the existing `event$` property instead of creating a separate one. But let me know if there was another intent for the `event$` other then emitting soemething to outside subscribers.